### PR TITLE
ListLogosWeek25

### PR DESCRIPTION
**Pull request by mrnschrmm**

### CODE REVIEW

File: Components/ListLogos/Partials/Image/index.twig [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        {{ image.src|resize(240, 135) }} 240w,
+        {{ image.src|resize(206, 116) }} 206w,
+        {{ image.src|resize(170, 96) }} 170w,
+        {{ image.src|resize(164, 92) }} 164w,
+        {{ image.src|resize(128, 72) }} 128w,
+        {{ image.src|resize(114, 64) }} 114w"
+      data-sizes="auto"
+      alt="{{ image.alt|e }}">
+    {% if item.title %}
+      <figcaption class="image-caption">{{ link.title|e }}</figcaption>
```
Comment (Qakulukiam) [inline Components/ListLogos/Partials/Image/index.twig]: item or link?

File: Components/ListLogos/_style.scss [hunk 2]
```diff
@@ -0,0 +1,100 @@
+[is='flynt-list-logos'] {
+  padding: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-desktop) {
+    padding: $component-spacing-desktop 0;
```
Comment (Qakulukiam) [inline Components/ListLogos/_style.scss]: there is now an additional variable `$component-spacing-tablet` that you can add here

File: Components/ListLogos/_style.scss [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  }
+
+  .preContent {
+    @include reset-content-margins;
+
+    margin-bottom: 24px;
+    text-align: center;
+
+    @media (min-width: $breakpoint-desktop) {
+      margin-bottom: 3rem;
```
Comment (Qakulukiam) [inline Components/ListLogos/_style.scss]: why rem here and px above?

File: Components/ListLogos/_style.scss [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    }
+  }
+
+  .list {
+    @include grid(2, center, 12px, 12px);
+    align-items: center;
+    list-style: none;
+    margin: 0;
+    padding: 0;
```
Comment (Qakulukiam) [inline Components/ListLogos/_style.scss]: you can use the `reset-list` mixin instead here

File: Components/ListLogos/_style.scss [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+    @media (min-width: 375px) {
+      @include grid(3, center,16px, 16px);
```
Comment (Qakulukiam) [inline Components/ListLogos/_style.scss]: missing space after comma

File: Components/ListLogos/_style.scss [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+  }
+
+  .media {
+    display: block;
+    height: 0;
+    padding-bottom: 70%;
+    position: relative;
+
+    @at-root .flyntComponent--card#{&} {
+      background-color: #fff;
```
Comment (Qakulukiam) [inline Components/ListLogos/_style.scss]: color could be background color variable

File: Components/ListLogos/_style.scss [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+      padding-bottom: 100%;
+    }
+
+    &--svg {
+      height: auto;
+      padding-bottom: unset;
```
Comment (Qakulukiam) [inline Components/ListLogos/_style.scss]: no support for this property in IE11, should be 0 instead

File: Components/ListLogos/Partials/Image/_style.scss [hunk 8]
```diff
@@ -0,0 +1,18 @@
+.image {
+  display: block;
+  height: auto;
+  margin: 0;
+  width: 100%;
+
+  .image-tag {
```
Comment (Qakulukiam) [inline Components/ListLogos/Partials/Image/_style.scss]: unnecessary nesting, selector could be `&-tag`

File: Components/ListLogos/index.twig [hunk 9]
```diff
@@ -0,0 +1,30 @@
+<div class="flyntComponent {{ options.card ? 'flyntComponent--card' }} {{ options.theme }}" is="flynt-list-logos">
+  <div class="container">
+    <div class="listLogos">
+      {% if preContent %}
+        <header class="preContent">
+          {{ preContent }}
```
Comment (Qakulukiam) [inline Components/ListLogos/index.twig]: missing escaping

File: Components/ListLogos/_style.scss [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+      .logo {
+        position: static;
+
+        @at-root .flyntComponent--card#{&} {
+          position: absolute;
+        }
+      }
+
+      .image {
```
Comment (Qakulukiam) [inline Components/ListLogos/_style.scss]: have you considered putting this into the partials where its other styles are?

File: Components/ListLogos/_style.scss (hunk 6 above)
Comment (mrnschrmm) [reply Components/ListLogos/_style.scss]: i added the background-light variable for the default theme ... the others have to be in any case white white white or #fff #fff #fff : > // i'm btw not able to chain the @at-root selector with a comma but i'm also not hurt by it.

**Comment by mrnschrmm** [comment]
> I’m not convinced it’s a good idea to allow the user to upload SVGs. They would be able to break a lot and there are no instructions telling them what to do to ensure they don’t.

i disabled the support in removing it from the allowed mime_types. i also think its actually a good way to keep the feature away from misuse but close enough to activate it on desperate demand.

**Pull request merged**